src/components/qrcode-scanner/QRCodeScanner.js

```diff
@@ -60,7 +60,7 @@ export default function QRCodeScanner({
     [setCameraEnabled]
   );
   useEffect(
-    () => dsRef.current.addOnCrossMagicBorderListener(onCrossMagicBorder),
+    () => dsRef.current?.addOnCrossMagicBorderListener(onCrossMagicBorder),
```

Comment:
it doesn't work on Android?

Reply:
Christian found a crash while having discover disabled / or enabled (don't remember) and that prevent it

Reply:
It is interesting. Can we check why?